### PR TITLE
Cryo 84

### PR DESCRIPTION
**Pull request by andypbarrett**
This PR is intended to get the `CRYO-84` branch to a state where it can be merged with `main`.  This will allow further updates to be addressed in CRYO-187.

Changes made are
- Remove unnecessary packages from environment.yml
- Replaced spaces with underscores in SMAP notebook filenames and change to lowercase
- Update original xarray notebook to fit tutorial style

### CODE REVIEW

File: notebooks/SMAP/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## SMAP Python Notebooks
+
+### Summary
+
+In this set of three tutorials we demonstrate how to search for, download and plot SMAP data. Tutorial 1 demonstrates how to search for and download SMAP data using the `earthaccess` library. The second tutorial demonstrates how to read in and plot the data downloaded in Tutorial 1. And Tutorial 3 provides information on the surface quality and retrieval quality flags.
```
Comment (asteiker) [inline notebooks/SMAP/README.md:5]: @andypbarrett Do we also want to reference the xarray notebook here?

File: notebooks/SMAP/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+### Summary
+
+In this set of three tutorials we demonstrate how to search for, download and plot SMAP data. Tutorial 1 demonstrates how to search for and download SMAP data using the `earthaccess` library. The second tutorial demonstrates how to read in and plot the data downloaded in Tutorial 1. And Tutorial 3 provides information on the surface quality and retrieval quality flags. 
+
+We use the [SMAP L3 Radiometer Global Daily 36 km EASE-Grid Soil Moisture, Version 8](https://nsidc.org/data/SPL3SMP/versions/8) data set as an example
+
+**NOTE** these notebooks are an updated version of the notebooks orginially published in this [repo](https://github.com/nsidc/smap_python_notebooks/tree/main). The notebooks are based on notebooks originally provided to NSIDC by Adam Purdy. Jennifer Roebuck of NSIDC applied the following updates:
+* Used `earthaccess` instead of `requests` for authentication, searching for and downloading the data. This reduced the code to just a few lines.
+* Used the latest version of SPL3SMP (version 8).
```
Comment (asteiker) [inline notebooks/SMAP/README.md]: The latest version is now 9. I will test with 9 and replace if everything works out of the box.

File: notebooks/SMAP/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+## SMAP Python Notebooks
+
+### Summary
+
+In this set of three tutorials we demonstrate how to search for, download and plot SMAP data. Tutorial 1 demonstrates how to search for and download SMAP data using the `earthaccess` library. The second tutorial demonstrates how to read in and plot the data downloaded in Tutorial 1. And Tutorial 3 provides information on the surface quality and retrieval quality flags. 
+
+We use the [SMAP L3 Radiometer Global Daily 36 km EASE-Grid Soil Moisture, Version 8](https://nsidc.org/data/SPL3SMP/versions/8) data set as an example
+
+**NOTE** these notebooks are an updated version of the notebooks orginially published in this [repo](https://github.com/nsidc/smap_python_notebooks/tree/main). The notebooks are based on notebooks originally provided to NSIDC by Adam Purdy. Jennifer Roebuck of NSIDC applied the following updates:
```
Comment (asteiker) [inline notebooks/SMAP/README.md:9]: We may need a follow on ticket to address this repo: Do we completely remove the repo (and whatever pointers we currently have to it on NSIDC's website) and/or update to point over here.

File: notebooks/SMAP/README.md (hunk 2 above)
Comment (asteiker) [reply notebooks/SMAP/README.md]: The v9 data are still being reprocessed and are not available over our example study area. I recommend we stick with v8 for now and can address the update under CRYO-72

File: notebooks/SMAP/03_smap_quality_flags.ipynb
Comment (asteiker) [inline notebooks/SMAP/03_smap_quality_flags.ipynb:1]: I received a matplot lib warning for each image: ```/tmp/ipykernel_478/3335087365.py:3: MatplotlibDeprecationWarning: The get_cmap function was deprecated in Matplotlib 3.7 and will be removed two minor releases later. Use ``matplotlib.colormaps[name]`` or ``matplotlib.colormaps.get_cmap(obj)`` instead.
  cax = ax.imshow((surf_flag_L3_P>>i)&1, cmap=plt.cm.get_cmap('bone', 2))```

**Review by asteiker — COMMENTED** [review]
@andypbarrett I just wanted to walk through the comments with you before merging to make sure we're capturing future work and whether we need to address the deprecation warning.

File: notebooks/SMAP/README.md (hunk 3 above)
Comment (andypbarrett) [reply notebooks/SMAP/README.md:9]: Probably a good idea to remove old repo.  Or remove references to it from notebooks and let original repo go stale.

File: notebooks/SMAP/03_smap_quality_flags.ipynb
Comment (andypbarrett) [reply notebooks/SMAP/03_smap_quality_flags.ipynb:1]: Let's create a separate issue to deal with this and address it in a different PR

File: notebooks/SMAP/README.md (hunk 3 above)
Comment (asteiker) [reply notebooks/SMAP/README.md:9]: Okay I will add this as a new DUE ticket.

**Pull request merged**